REST/PowerShell/Variables/GetVariableInProject.ps1

```diff
@@ -0,0 +1,106 @@
+Function Get-OctopusProject
```

Comment:
Can we make the script Space aware? See https://octopus.com/docs/octopus-rest-api/examples/create-and-deploy-a-release#create-and-deploy-a-release for an example